server.js

```diff
@@ -1,27 +1,23 @@
-const express = require('express')
-const cors = require('cors');
-const inventoriesRoutes = require('./routes/inventoriesRoutes');
-const app = express()
-const port = 3000
+const express = require("express");
+const cors = require("cors");
+const inventoriesRoutes = require("./routes/inventoriesRoutes");
+const warehouseRoutes = require("./routes/warehouseRoutes");
+const app = express();
+const port = 3000;
 
 // Middleware
-app.use(express.json()) 
-app.use(cors()); 
-app.use(express.static('public'))
+app.use(express.json());
+app.use(cors());
+app.use(express.static("public"));
 
-
-
-
-
-app.get('/', (req, res) => {
-  res.send('Hello World!')
+app.get("/", (req, res) => {
+  res.send("Hello World!");
 });
 
-
-// Routes 
-app.use('/inventories/:warehouse_id', inventoriesRoutes);
-
+// Routes
+app.use("/inventories/", inventoriesRoutes);
+app.use("/warehouses/", warehouseRoutes);
```

Comment:
In your warehouseController, you get the warehouse_id from the req.params.id but in this server js file your route is not dynamically allowing a params in the route, which means req.params.id will be undefined  
and if you check your ticket on Jira you gonna see the expected route 

Fix 
```
server.js
app.use("/warehouses/:id", warehouseRoutes);

warehouseController.js
const warehouseId = parseInt(req.params.id);
```